Angular: add TypeScript Support topic (#5658)
DevExpress/devextreme-documentation: concepts/40 Angular Components/70 TypeScript Support/00 TypeScript Support.md

> TypeScript is a [primary language](https://angular.io/guide/typescript-configuration) for Angular development. This help topic outlines additional TypeScript specifics of developing with DevExtreme components.

Reviewer: ```suggestion
TypeScript is a [primary language](https://angular.io/guide/typescript-configuration) for Angular development. This help topic outlines additional TypeScript specifics related to development with DevExtreme components. 
```
